heroku/templates/src/main/java/package/config/_HerokuDatabaseConfiguration.java

```diff
@@ -32,32 +32,22 @@ public void setEnvironment(Environment environment) {
     public DataSource dataSource(<% if (hibernateCache == 'hazelcast') { %>CacheManager cacheManager<% } %>) {
         log.debug("Configuring Heroku Datasource");
 
-        String herokuUrl = propertyResolver.getProperty("heroku-url");
+        String herokuUrl = System.getenv("JDBC_DATABASE_URL");
```

Comment:
Do you mean I should call `propertyResolver.getProperty("url")` and then set the Spring property to the env var in the configuration (or Procfile)?